### DISCUSSION TITLE
Name Ideas for the English Learning Platform

### DISCUSSION THREAD

**Discussion by ebrarkiziloglu** [bounswe/bounswe2024group5#280] [Ideas]
We need to determine the name of our platform. Let's suggest cool names for it!

**Comment by fahreddinozcan** [comment]
lanquiz

**Comment by ebrarkiziloglu** [comment]
Learnify

**Comment by ramazanoacar** [comment]
lingoduo

**Comment by odenizddd** [comment]
<details>

<summary>Other ideas for inspiration.</summary>

Repeto, WoW (World of Words), Kelam, Lafla, Worden, Upword, WordHub, Kelimatör, Wordmatic, Wordistan, Tekrar, LevelUp, Prose, Litero, Vochubulary, Quizzard, BlaBla

</details>

**Comment by odenizddd** [comment]
Repeto

**Comment by odenizddd** [comment]
Quizzard

**Comment by odenizddd** [comment]
WordHub

**Comment by ramazanoacar** [comment]
Kelam da çok iyiymiş

**Comment by ramazanoacar** [comment]
Quizzard tüm oyları alarak isim hakkını kazandı
